packages/app/src/systems/Home/components/UserAddressCard/UserAddressCard.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box } from '@fuel-ui/react';
+
+import { UserAddressCard } from './UserAddressCard';
+
+export default {
+  component: UserAddressCard,
+  title: 'Home/Components/UserAddressCard',
+};
+
+export const Usage = () => (
+  <Box css={{ width: 300 }}>
+    <UserAddressCard account="fuel1auahknz6mjuu0am034mlggh55f0tgp9j7fkzrc6xl48zuy5zv7vqa07n30" />
```

Comment:
`fuel1auahknz6mjuu0am034mlggh55f0tgp9j7fkzrc6xl48zuy5zv7vqa07n30` is informed, but screen is missing the last `0`, showing only: 
`fuel1auahknz6mjuu0am034mlggh55f0tgp9j7fkzrc6xl48zuy5zv7vqa07n3` 

is this expected?